plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -254,6 +293,17 @@ def createProject(self, namespace, arguments):
 
         return project
 
+    '''
+    @param arguments Attributes of the project
+    '''
+    def getOptionsWithValue(self, arguments):
+        retArguments = dict()
```

Comment:
```suggestion
        ret_arguments = dict()
```
(same for the other usages)